lms/djangoapps/instructor/enrollment_report.py

```diff
@@ -0,0 +1,56 @@
+"""
+Defines abstract class for the Enrollment Reports.
+"""
+from instructor.enrollment_report_provider import EnrollmentReportProvider
+from django.contrib.auth.models import User
+import collections
+import json
+
+
+class BaseEnrollmentReportProvider(EnrollmentReportProvider):
```

Comment:
If you do keep both classes, rename `EnrollmentReportProvider` to `AbstractEnrollmentReportProvider` and keep both classes in the same file.